src/types/libdef/npm-custom/@testing-library/react_v11.x.x.js

```diff
@@ -1,11 +1,170 @@
+// This was copied from https://github.com/flow-typed/flow-typed/blob/2997a88721d3ed0057cdb87e363c5d01b7372998/definitions/npm/%40testing-library/react_v11.x.x/flow_v0.104.x-/react_v11.x.x.js
+// And adapted for our version of Flow
```

Comment:
The version in flow-typed isn't for Flow > 104. The good thing is that it works well for our version of Flow too!